Modelica/Utilities/Internal.mo

```diff
@@ -50,6 +50,36 @@ This model is documented at
 </html>"));
 
   end PartialShape;
+  
+    model PartialVector "Interface for 3D animation of a vector quantity (force, torque etc)"
+      import Modelica.Mechanics.MultiBody.Types;
+      import Modelica.Mechanics.MultiBody.Frames;
+
+      input Frames.Orientation R=Frames.nullRotation()
+        "Orientation object to rotate the world frame into the vector frame" annotation(Dialog);
+      input Modelica.SIunits.Position r[3]={0,0,0}
+        "Position vector from origin of world frame to origin of vector, resolved in world frame" annotation(Dialog);
+      input Real coordinates[3]={0,0,0}
+        "Coordinates of the vector resolved in vector frame" annotation(Dialog);
+      input Types.Color color=Types.Defaults.ArrowColor
+        "Color of vector" annotation(Dialog(colorSelector=true));
+      input Types.SpecularCoefficient specularCoefficient = 0.7
+        "Material property describing the reflecting of ambient light (= 0 means, that light is completely absorbed)"
+        annotation(Dialog);
+      input Boolean pushing=false "= true, if the vector is pointing towards the origin" annotation(Dialog);
+      input Boolean doubleArrow=false "= true, if the arrow has two heads after each other (pointing in the same direction)" annotation(Dialog);
```

Comment:
I do not like the names `pushing` and `doubleArrow` that much.

1. Should we rename `pushing` to something like `inwardArrow` or `arrowDirection` or ...?
1. `doubleArrow` could also mean :left_right_arrow:. Should we rename it to `twoHeadedArrow` to avoid disambiguation?

Reply:
> 1. pushing is associated with force arrows and not general enough. Should we rename pushing to something like inwardArrow or arrowDirection or ...?

Actually, the vector is given by its coordinates having the origin in the "vector frame". So generally, the vector is always pointing out of the vector's frame, i.e. the vector's tail is identical to origin and vector's head it given by its coordinates. What is changed by "pushing" is the head/tail assignment. So that input's name could somehow addres this. But I have no good idea.
In `Modelica.Mechanics.MultiBody.Sensors.CutForce` there is used `positiveSign` but it is not suitable here. 


> 2. doubleArrow could also mean ↔️. Should we rename it to twoHeadedArrow to avoid disambiguation?

or "doubleArrowHead" or simply "doubleHead"...

Reply:
Hm, is `pushing` still used after all or is it a left-over from development?

Reply:
It is still used.

Reply:
What about "tailAtOrigin" or "headByCoordinates" ...

Reply:
Why not `headAtOrigin`?

Reply:
I think "headAtOrigin" would work, and similarly twoHeadedArrow.

Reply:
@HansOlsson Shall I change it?

Reply:
> Why not headAtOrigin?

Due to original setting of `pushing=false` - so to my understanding the tail was at origin. But no problem about it.

Reply:
But "headAtOrigin" and "pushing" should mean the same, right?
I.e. you are pushing at the origin when the head is there.

Reply:
Actually yes. You're right.